docs/src/modules/utils/postProcessImport.test.ts

```diff
@@ -1,4 +1,3 @@
-import { expect } from 'chai';
```

Comment:
This is basically the main "noisy" change - relying on the bundled `chai` imported from `vitest/globals`.